pkg/auth/audit/audit.go

```diff
@@ -76,6 +80,16 @@ func getUserInfo(req *http.Request) *User {
 	}
 }
 
+func getUserNameForBasicLogin(body []byte) (string, bool) {
+	input := &v32.BasicLogin{}
+	err := json.Unmarshal(body, input)
+	if err != nil {
+		logrus.Debugf("Not a BasicLogin input, cannot add login info to audit log: %v", err)
```

Comment:
I don't think we are going to see an error here, the struct will just continue to be empty if no fields match from the input.